src/prop-converter.js

```diff
@@ -47,14 +47,6 @@ declare var process: {
   },
 };
 
-function isProduction(): boolean {
-  try {
-    return process.env.NODE_ENV === 'production';
-  } catch (e) {
-    return false;
-  }
-}
```

Comment:
Nice, happy we get to kill this. Is the idea that rollup automatically replaces `process.env` values?

Reply:
i figured the function doesnt get stripped, whereas the conditional block would be?

Reply:
> Nice, happy we get to kill this. Is the idea that rollup automatically replaces process.env values?

Yeah, that's done by `rollup-plugin-replace` for umd builds, cjs and es stays untouched in this regard and will most likely be replaced by consumer's bundler (webpack or whatever). This is common practice to branch code depending on the environment, react itself does this so users of `react-flip-move` should have their replacing plugins already in place.

And as mentioned branchinc code cant be dropped because of this function call, whereas when this gets replaced to i.e. smth like:
```js
if ("production" === "development") {
  // some dev only warnings
}
```

UglifyJs will be able to actually recognize it as `if (false)` and will drop the entire block as its unreachable.

Reply:
Cool! Awesome, thanks. Yeah, was sad that we lost dead code elimination with the try/catch, but it was necessary with the old infra.